src/main/resources/com/adobe/epubcheck/schema/30/package-30.sch

```diff
@@ -295,5 +295,62 @@
     </pattern>
     
     <include href="./mod/id-unique.sch"/>
-
-</schema>
+	
+	<!-- EPUB 3.2 New Checks -->
+	
+	<pattern id="opf.spine.duplicate.refs">
+		<rule context="opf:itemref[@idref = preceding-sibling::opf:itemref/@idref]">
+			<report test=".">Itemref refers to the same manifest entry as a previous itemref</report>
+		</rule>
+	</pattern>
+	
+	<pattern id="opf.subject.authority-term">
+		<rule context="opf:metadata/dc:subject">
+			<let name="id" value="./@id"/>
+			<let name="authority" value="//opf:meta[@property='authority'][substring(@refines, 2) = $id]"/>
```

Comment:
Yikes, yes. Looking through the old tests, there are lots that won't check what they're supposed to if there is any whitespace around them. I'm going through and updating them all now to add normalize-space where needed.

Reply:
It's actually a problem all over with attribute tests. XML processors will normalize whitespace in attributes, but there are lots of tests where we're not taking that into account. For example, `id="    foo    "`  is valid, but `unique-identifier="foo"` won't properly test it, and vice versa. In some cases, like the previous, it won't matter because an exact match is required, but overall the tests are a combination of being too brittle and too strict.